Ginger/GingerCoreCommon/External/Configurations/AccessibilityRuleData.cs

```diff
@@ -14,10 +14,19 @@ public class AccessibilityRuleData : RepositoryItemBase, INotifyPropertyChanged
         public event PropertyChangedEventHandler PropertyChanged;
 
         private bool mActive = false;
+        /// <summary>
+        /// Gets or sets the Active associated with the accessibility rule.
+        /// </summary>
         [IsSerializedForLocalRepository]
         public bool Active { get { return mActive; } set { if (mActive != value) { mActive = value; OnPropertyChanged(nameof(Active)); } } }
+        /// <summary>
+        /// Gets or sets the Rule Id associated with the accessibility rule.
+        /// </summary>
         [IsSerializedForLocalRepository]
         public string RuleID { get; set; }
```

Comment:
Please define field for this and then use it in the property. Same way as done for Active

Reply:
updated